.github/workflows/tests-benchmark.yml

```diff
@@ -0,0 +1,45 @@
+name: "Integration Tests"
```

Comment:
I'd rather rename the `name` or the file.
name is integration tests and the file is tests benchmark, which one is the correct one?